packages/combobox/src/Combobox.ts

```diff
@@ -508,7 +508,9 @@ export class Combobox extends Textfield {
         }
         if (changed.has('activeDescendant')) {
             if (changed.get('activeDescendant')) {
-                (changed.get('activeDescendant') as MenuItem).focused = false;
+                (
+                    changed.get('activeDescendant') as unknown as MenuItem
+                ).focused = false;
```

Comment:
I'm sure looking up 'activeDescendant' twice isn't expensive, but doing it only once would preclude typos with the key:

```
const descendant = changed.get('activeDescendant') as unknown as MenuItem | undefined;
if (descendant) {
  descendant.focused = false;
}
```